helix-term/src/commands.rs

```diff
@@ -3832,6 +3833,15 @@ fn completion(cx: &mut Context) {
 
     let trigger_offset = cursor;
 
+    let left_line = get_line_left(doc.text(), cursor);
```

Comment:
The name is too vague; to me it implies that it is a line to the left of something.